src/SecTester.Scan/ScanSettings.cs

```diff
@@ -0,0 +1,199 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using SecTester.Scan.Models;
+using SecTester.Scan.Target;
+
+namespace SecTester.Scan;
+
+public record ScanSettings : ScanSettingsOptions
+{
+  private const int MinNameLength = 1;
+  private const int MaxNameLength = 200;
+  private const int MinPoolSize = 1;
+  private const int MaxPoolSize = 50;
+  private const bool DefaultSmart = true;
+  private const bool DefaultSkipStaticParams = true;
+  private const int DefaultPoolSize = 10;
```

Comment:
Why do you introduce these static fields? Why don't assign default values directly to the accessible fields?